### PR TITLE
Minor grammar and style changes

### PR DESCRIPTION
**Pull request by anton-intel**
Just little nitpicks.

### CODE REVIEW

**Comment by lgirdwood** [comment]
@anton-intel btw, feel free to merge grammar and style fixes. I cant add any value in review.

**Review by deb-intel — APPROVED** [review]

**Comment by marc-hb** [comment]
@anton-intel please add the original author as a reviewer on fixes like this. For at least two reasons:
- Non-native speakers can learn from you
- They can also make sure any English mistake they made is not technically misinterpreted and amplified by you

GitHub has a useful "blame" button where you can see who wrote each line.

File: getting_started/intel_debug/introduction.rst [hunk 1]
```diff
@@ -142,60 +142,61 @@ wrong file is selected at any of the three layers.
...
   <https://chromium.googlesource.com/chromiumos/docs/+/HEAD/developer_guide.md>`_.
   The ``cros_sdk`` is a complete environment that lets you modify
-  anything in ChromeOS and even build an entire system image. The
-  ``cros_sdk`` requires significant disk space and there is some
-  learning effort if you're not already familiar with Gentoo's build
-  system "Portage"; especially for the Linux kernel.
+  anything in Chrome OS and even build an entire system image. The
+  ``cros_sdk`` requires significant disk space and some learning
+  effort if you are not already familiar with Portage, a package
+  management system in Gentoo, and Linux kernel build process.
```
Comment (marc-hb) [inline getting_started/intel_debug/introduction.rst:172]: This is not what I tried to say. I meant: being familiar with Portage and especially with building the kernel in Portage.

Portage is also a package management for Gentoo but in Chrome OS the package management part is not really used, it's really the build system that is.

**Review by marc-hb — COMMENTED** [review]
Looks great except for the Portage bit. It's not very important so nothing urgent.

**Comment by greg-intel** [comment]
I would actually LIKE to be on all README and documentation tweaks.
For one thing, I'm still learning the code base, so things don't have intuitive clarity for me.
Also, I'm also into documentation being as clear as possible.
Hopefully I don't regret saying this.

**Comment by marc-hb** [comment]
> I would actually LIKE to be on all README and documentation tweaks.

Thanks! we can always use more reviewers (and native speakers).
@greg-intel please submit a PR where you add yourself to the CODEOWNERS file and let's discuss it there, it's off-topic here.

> Hopefully I don't regret saying this.

There's always some "unsubscribe" button not far...

File: getting_started/intel_debug/introduction.rst (hunk 1 above)
Comment (anton-intel) [reply getting_started/intel_debug/introduction.rst:172]: @marc-hb, I created PR #387 for this correction.

**Pull request merged**